website/content/docs/platform/servicenow/configuration.mdx

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+page_title: Vault Credential Resolver Configuration
+description: This section documents the configurables for the Vault Credential Resolver.
+---
+
+# MID server properties
+
+The following properties are supported by the Vault Credential Resolver:
```

Comment:
We could link to the servicenow properties page here, something like

```suggestion
The following [properties](https://docs.servicenow.com/bundle/quebec-servicenow-platform/page/product/mid-server/reference/r_MIDServerProperties.html#t_SetMIDServerProperties) are supported by the Vault Credential Resolver:
```
Though maybe it's obvious what properties are for anyone working with ServiceNow when they get here.

Reply:
Thanks, good idea